core/app/lib/uffizzi_core/concerns/models/deployment.rb

```diff
@@ -27,7 +27,7 @@ module UffizziCore::Concerns::Models::Deployment
 
     validates :kind, presence: true
 
-    enumerize :creation_source, in: [:manual, :continuous_preview, :compose_file_manual, :compose_file_continuous_preview],
+    enumerize :creation_source, in: [:manual, :continuous_preview, :compose_file_manual, :compose_file_continuous_preview, :github_actions],
```

Comment:
I think we should add `github_actions` only to `uffizzi_platform`. Gem (uffizzi) should not know anything about github